src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListBoxTests.cs

```diff
@@ -1382,6 +1382,30 @@ public static IEnumerable<object[]> HorizontalExtent_Set_TestData()
             }
         }
 
+        [WinFormsFact]
+        public void ListBox_Height_IsEqualDefaultHeight_IfDefaultFontIsChanged()
+        {
+            var oldApplicationDefaultFont = Application.DefaultFont;
+            using Font font = new Font("Times New Roman", 12);
+            using TestListBox listBox = new TestListBox();
+            var applicationTestAccessor = typeof(Application).TestAccessor().Dynamic;
+
+            try
+            {
+                applicationTestAccessor.s_defaultFont = font;
+                Assert.NotEqual(oldApplicationDefaultFont, Application.DefaultFont);
+
+                var rowDefaultHeight = listBox.GetDefaultListBoxItemHeight();
```

Comment:
Since `GetDefaultListBoxItemHeight` is used only here, perhaps it can be inlined and then TestListBox can be removed?
```suggestion
                int rowDefaultHeight = listBox.TestAccessor().Dynamic.DefaultListBoxItemHeight;
```

Reply:
Yes. Now than you've pointed it out, I understood that I expected this class to be more useful.